packages/http-server/src/AjvValidator.ts

```diff
@@ -182,6 +183,16 @@ export class AjvValidator<Request = unknown> {
 		}
 	}
 
+	private registerPlugins() {
+		this.options.plugins.forEach(p => {
```

Comment:
`this.options.plugins` may be undefined, better to add a check there

Reply:
Sure, added optional chaining